sdcm/cluster_gce.py

```diff
@@ -224,6 +224,15 @@ def _get_persistent_disk_struct(self, name, disk_size, disk_type='pd-ssd', dc_id
                 "autoDelete": True}
 
     def _create_instance(self, node_index, dc_idx):
+        # if size of disk is larget than 80G, then
+        # change the timeout of job completion to default * 3.
+        gce_job_default_timeout = None
+        if self._gce_image_size and int(self._gce_image_size) > 80:
+            gce_job_default_timeout = self._gce_services[dc_idx].connection.timeout
+            if hasattr(self._gce_services[dc_idx].connection, 'timeout'):
```

Comment:
BTW: why do you check for attr existence here? Can it be a case when there is no timeout attribute on the connection object?

Reply:
it is some kind of  insurance that this attribute is present

Reply:
if you didn't meet the case when this attr mysteriously evaporate please remove this check